common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -2810,7 +2810,7 @@ schematic = (function() {
 			    // for each requested freq, interpolate response value
 			    for (var k = 1; k < flist.length; k++) {
 				var f = flist[k];
-				var v = interpolate(f,x_values,values);
+				var v = interpolate(f,x_values,values);  //xss-lint: disable=javascript-interpolate
```

Comment:
I don’t know what these values are and where they come from to know if this is safe. Are you clear that it is?

Reply:
This interpolate method is domain specific and doesn't directly reflect or store data to users.  It is filtering circuit timings for the simulation.  The timings could be manipulated, but display of them would be captured by the linter elsewhere.